spec/change_sets/ephemera_folder_change_set_spec.rb

```diff
@@ -22,6 +22,37 @@
     end
   end
 
+  describe "#date_range" do
+    it "can set it" do
+      change_set.prepopulate!
+      change_set.validate(date_range_form_attributes: { start: "2017", end: "2018" })
+      change_set.sync
+      expect(change_set.model.date_range.first.start).to eq ["2017"]
+    end
+    it "validates" do
+      change_set.prepopulate!
+      result = change_set.validate(date_range_form_attributes: { start: "abcd", end: "2018" })
+      expect(result).to eq false
+    end
+    it "validates that the start is before the end" do
+      change_set.prepopulate!
+      result = change_set.validate(date_range_form_attributes: { start: "2018", end: "2017" })
+      expect(result).to eq false
+    end
+    it "has a default" do
+      change_set.prepopulate!
+      expect(change_set.date_range_form.start).to be_nil
+      expect(change_set.date_range_form.required?(:start)).to eq false
+    end
+    context "when there's a date range set" do
+      it "sets it single-valued appropriately" do
+        change_set = described_class.new(FactoryGirl.build(:ephemera_folder, date_range: DateRange.new(start: "2017", end: "2018")))
+        change_set.prepopulate!
+        expect(change_set.date_range_form.start).to eq "2017"
+      end
```

Comment:
What behavior is expected if one of the values is given but the other is blank?

Reply:
I figured it would be fine, but we should probably make it required.

Regarding `date_created`, I wonder if we should keep it around for the weird dates (thinks with question marks and such)

Reply:
What is the indexing logic for date_created? If that is where single dates live it should be possible to search for them alongside date ranges.

Reply:
It's just indexed the same as everything else - into a `date_tesim` field. I don't think you can search for it in Figgy, atm.

Reply:
Maybe we should create a ticket for the old date field and ask the Ephemera people what they want?

Reply:
New ticket sounds good!

Reply:
Created #313 and added a new validation to make sure if start is set then end must be too and vice versa.